inc/commondbtm.class.php

```diff
@@ -5167,10 +5167,11 @@ static function generateLinkContents($link, CommonDBTM $item) {
     *                        - content_field (default content) the field who receive the main text
     *                                                          (with images)
     *                        - name (default filename) name of the HTML input containing files
+    * @param mixed $date     Date to set on document_items
     *
     * @return array the input param transformed
    **/
-   function addFiles(array $input, $options = []) {
+   function addFiles(array $input, $options = [], $date = null) {
```

Comment:
instead of adding a new parameter, can we use the options array instead ?

Reply:
Done 611dc7e58d.